app/build.gradle

```diff
@@ -110,97 +150,107 @@ android {
             events 'passed', 'skipped', 'failed', 'standardOut', 'standardError'
             exceptionFormat 'full'
         }
+        //Print the test result summary
+        afterSuite { desc, result ->
+            if (!desc.parent) { // will match the outermost suite
+                println "Myntra Test Results: ${result.resultType} (${result.testCount} tests, ${result.successfulTestCount} successes, ${result.failedTestCount} failures, ${result.skippedTestCount} skipped)"
+            }
+        }
     }
 
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
 
     //PLAY SERVICES
-    compile libraries.playServicesAd
-    compile libraries.playServicesGcm
-    compile libraries.playServicesMaps
-    compile libraries.playServicesLocation
+    implementation libraries.playServicesAd
+    implementation libraries.playServicesGcm
+    implementation libraries.playServicesMaps
+    implementation libraries.playServicesLocation
 
     //SUPPORT LIBS
-    compile libraries.supportV13
-    compile libraries.supportDesign
-    compile libraries.supportCardView
-    compile libraries.supportMultiDex
-    compile libraries.supportAppCompat
-    compile libraries.supportRecyclerView
-
-    compile libraries.eventBus
-    compile libraries.butterKnife
+    implementation libraries.supportV13
+    implementation libraries.supportDesign
+    implementation libraries.supportCardView
+    implementation libraries.supportMultiDex
+    implementation libraries.supportAppCompat
+    implementation libraries.supportRecyclerView
+
+    implementation libraries.eventBus
+    implementation libraries.butterKnife
```

Comment:
Neat.  Am I reading this right?:
>New api, implementation, compileOnly and runtimeOnly configurations allow libraries to not always leak their dependencies to their consumers.
https://developer.android.com/studio/preview/features/new-android-plugin.html

Just curious how this is different from `compile`, if you happen to know.